02_commands.html

```diff
@@ -1,206 +1,524 @@
+      </tr>
+      <tr>
+        <td>GET</td>
+        <td>/session/{sessionId}/element/{elementId}/enabled</td>
+        <td><a>Is Enabled</a></td>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{sessionId}/execute</td>
+        <td><a>Execute Script</a></td>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{sessionId}/execute_async</td>
+        <td><a>Execute Async Script</a></td>
+      </tr>
+      <tr>
+        <td>GET</td>
+        <td>/session/{sessionId}/cookie</td>
+        <td><a>Get Cookie</a></td>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{sessionId}/cookie</td>
+        <td><a>Add Cookie</a></td>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{sessionId}/timeouts</td>
+        <td><a>Set Timeouts</a></td>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{sessionId}/actions</td>
+        <td><a>Actions</a></td>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{sessionId}/element/{elementId}/click</td>
+        <td><a>Element Click</a></td>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{sessionId}/element/{elementId}/tap</td>
+        <td><a>Element Tap</a></td>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{sessionId}/element/{elementId}/clear</td>
+        <td><a>Element Clear</a></td>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{sessionId}/element/{elementId}/sendKeys</td>
+        <td><a>Element Send Keys</a></td>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{sessionId}/dismiss_alert</td>
+        <td><a>Dismiss Alert</a></td>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{sessionId}/accept_alert</td>
+        <td><a>Accept Alert</a></td>
+      </tr>
+      <tr>
+        <td>GET</td>
+        <td>/session/{sessionId}/alert_text</td>
+        <td><a>Get Alert Text</a></td>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{sessionId}/alert_text</td>
+        <td><a>Send Alert Text</a></td>
+      </tr>
+      <tr>
+        <td>GET</td>
+        <td>/session/{sessionId}/screenshot</td>
+        <td><a>Take Screenshot</a></td>
+      </tr>
+    </table>
+  </section>
+
+  <section>
+    <h3>Handling Errors</h3>
+    <p>Errors are represented in the WebDriver protocol with a HTTP response with a HTTP status in the 4xx or 5xx range and a JSON body containing details of the error. This JSON body has two fields; <code>error</code>, containing a string indicating the error type and <code>error</code> containing an implementation-defined string containing a human readable description of the kind of error that occured.</p>
+
+    <aside class="example">
+      <p>A <code>DELETE</code> request to <code>/session/1234</code>, where <code>1234</code> is not the id of a currently active session would return a HTTP <code>404</code> response with a body of the form:</p>
```

Comment:
I think this is OK? It's non-normative anyway and "of the form" was supposed to be the clue that the structure is important but the details aren't.